sdk/neonpostgres/Azure.ResourceManager.NeonPostgres/src/Generated/BranchData.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.ResourceManager.Models;
+using Azure.ResourceManager.NeonPostgres.Models;
+
+namespace Azure.ResourceManager.NeonPostgres
+{
+    /// <summary>
+    /// A class representing the Branch data model.
+    /// The Branch resource type.
+    /// </summary>
+    public partial class BranchData : ResourceData
```

Comment:
I saw you have a PR to update the client.tsp: https://github.com/Azure/azure-rest-api-specs/pull/34043
but the name here is not changed.
Could we update your commit to point to the head commit of your PR and regenerate everything here?

Reply:
Done